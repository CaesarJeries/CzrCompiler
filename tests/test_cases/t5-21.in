void main() {
    byte a = 255b + 1b;
    printi(a);
    print("\n");
    a = a - 2b;
    printi(a);
    print("\n");
}