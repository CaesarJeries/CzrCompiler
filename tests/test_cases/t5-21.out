Loaded: ./exceptions.s
0
254
